docker/sshd/etc/teleport.yaml

```diff
@@ -0,0 +1,19 @@
+teleport:
+  nodename: luna.teleport
+  data_dir: /var/lib/teleport
+  log:
+    output: stderr
+    severity: INFO
+auth_service:
+  enabled: "yes"
+ssh_service:
+  enabled: "yes"
+  labels:
+    env: example
+  commands:
+  - name: hostname
+    command: [hostname]
+    period: 1m0s
+proxy_service:
+  enabled: "yes"
+  public_addr: ["proxy.luna.teleport"]
```

Comment:
Consider changing to `proxy.luna.teleport:443` and setting `listen_addr: 0.0.0.0:443` as detailed in a comment below